skrf/network.py

```diff
@@ -336,6 +338,10 @@ def __init__(self, file=None, name=None, comments=None, f_unit=None, **kwargs):
             its a str
         comments : str
             Comments associated with the Network
+        s_def : str
```

Comment:
when the  arg  can only be a  value  out of a list, usually you list all options (when possible)
`s_def :str` -> `s_def : ['power','pseudo']`

Reply:
OK